influxql/iterator.gen.go.tmpl

```diff
@@ -1271,3 +1273,9 @@ func (enc *IteratorEncoder) encodeStats(stats IteratorStats) error {
 }
 
 {{end}}
+
```

Comment:
Unless this is being templated, it should probably just be moved to `iterator.go`.